styles.css

```diff
@@ -1,4 +1,4 @@
-* {
+  * {
```

Comment:
I think this is an unnecessary space but you can correct it later, just a heads up.

Reply:
thank you